presto-product-tests/src/main/java/com/facebook/presto/tests/ImmutableLdapObjectDefinitions.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.google.common.collect.ImmutableMap;
+import com.teradata.tempto.fulfillment.ldap.LdapObjectDefinition;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import static java.lang.String.format;
+
+public final class ImmutableLdapObjectDefinitions
+{
+    private static final String domain = "dc=presto,dc=testldap,dc=com";
+    private static final String americaDN = format("ou=America,%s", domain);
+    private static final String asiaDN = format("ou=Asia,%s", domain);
+    private static final String password = "LDAPPass123";
+
+    private ImmutableLdapObjectDefinitions() {}
+
+    private static LdapObjectDefinition buildLdapOrganizationObject(String id, String distinguishedName, String unit)
+    {
+        return LdapObjectDefinition.builder(id)
+                .setDistinguishedName(distinguishedName)
+                .setAttributes(ImmutableMap.of("ou", unit))
+                .addObjectClasses(Arrays.asList("top", "organizationalUnit"))
+                .build();
+    }
+
+    private static LdapObjectDefinition buildLdapGroupObject(String groupName, String memberName)
+    {
+        return LdapObjectDefinition.builder(groupName)
+                .setDistinguishedName(format("cn=%s,%s", groupName, americaDN))
+                .setAttributes(ImmutableMap.of(
+                        "cn", groupName,
+                        "member", format("uid=%s,%s", memberName, asiaDN)))
+                .addObjectClasses(Arrays.asList("groupOfNames"))
```

Comment:
I should have added the comment on line-42. I mean this method 
private static LdapObjectDefinition buildLdapGroupObject(String groupName, String memberName)

This will create Group Object which only belong to America and user to Asia